pom.xml

```diff
@@ -127,6 +127,7 @@
         <error_prone_annotations.version>2.5.1</error_prone_annotations.version>
         <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
         <janino.version>3.1.6</janino.version>
+        <spring-web.version>5.3.12</spring-web.version>
```

Comment:
Is it necessary to introduce a new jar package?
There are the same network operations in the HTTP task.

Reply:
I have finished the revision.